src/test/scala/com/julienvey/trello/CardIt.scala

```diff
@@ -50,6 +53,15 @@ class CardIt extends FunSpec with Matchers {
       }
       thrown.getMessage should include("not found")
     }
+
+    it("assigns label to card") {
+      val cardId = CardGetITCase.CARD_ID
+      val label = randomUUID().toString
```

Comment:
I will merge the current version, but can you please update the test to create a separate card each time?
I feel that would make the test more resilient - say, if there are some potential limitations on how many labels a card can have, etc.